addons/mass_mailing/static/src/js/mass_mailing_html_field.js

```diff
@@ -230,15 +232,19 @@ export class MassMailingHtmlField extends HtmlField {
             }
             this.onIframeUpdated();
         });
-
-        $snippetsSideBar.on('click', '.o_mobile_preview_btn', () => {
+        const $previewBtn = $snippetsSideBar.find('.o_mobile_preview_btn');
+        $previewBtn.off('click');
+        $previewBtn.on('click', () => {
+            $previewBtn.prop('disabled', true); // Prevent double execution when double-clicking on the button
             let mailingHtml = new DOMParser().parseFromString(this.wysiwyg.getValue(), 'text/html');
             [...mailingHtml.querySelectorAll('a')].forEach(el => {
                 el.style.setProperty('pointer-events', 'none');
             });
             this.mobilePreview = this.dialog.add(MassMailingMobilePreviewDialog, {
                 title: this.env._t("Mobile Preview"),
                 preview: mailingHtml.body.innerHTML,
+            }, {
+                onClose: () => $previewBtn.prop('disabled', false),
```

Comment:
can't we use removeAttr instead ? (so the attribute is not in the DOM anymore)

Reply:
Good remark but after reading some posts and the following doc, it seems that using prop is the correct way as the attribute represents the initial value while the prop the current value (so modifying the attribute afterwards could not change the property value):
https://jquery.com/upgrade-guide/3.0/#feature-new-signature-for-jquery-get-and-jquery-post
What do you think ?